src/message/action.js

```diff
@@ -917,8 +919,22 @@ function Originate() {
  * @property {String} StateInterface Optional, State interface
  * @augments Action
  */
-function QueueAdd() {
+function QueueAdd(queue, interface, memberName, penalty, stateInterface) {
 	QueueAdd.super_.call(this, 'QueueAdd');
+    	this.set('queue', queue);
+    	this.set('interface', interface);
```

Comment:
**interface** is a reserved keyword, let's change the name, I'll be happy to merge :)